detection_rules/etc/example_test_config.yaml

```diff
@@ -0,0 +1,39 @@
+
+# set the environment variable DETECTION_RULES_TEST_CONFIG
+
```

Comment:
We may need to add another hook to bypass schema validation for timeline templates. Currently users can not use this feature because our definitions hardcode id's and titles. 

XREF: https://elasticstack.slack.com/archives/C016E72DWDS/p1711635850560659

Reply:
That would be inconsistent with this approach

- built in rules will always be compatible
- if it is a custom rule, they can bypass the test